_tutorials/new-component/create-a-new-component-part-2.md

```diff
@@ -162,15 +163,19 @@ The first step is to install each component we want to use via [Bower](https://b
 
 With Bower configured, we can now run:
 <pre><code class="o-syntax-highlight--bash">bower install o-colors@^5.0.0 o-spacing@^2.0.0 o-typography@^6.0.0 o-buttons@^6.0.0 --save</code></pre>
+=======
+The first step is to install each component we want to use via [npm](https://npmjs.com/):
+<pre><code class="o-syntax-highlight--bash">npm install --save '@financial-times/o-colors@^5.0.0' '@financial-times/o-spacing@^2.0.0' '@financial-times/o-typography@^6.0.0' '@financial-times/o-buttons@^6.0.0'</code></pre>
+>>>>>>> v2.0
 
-You should now have a `bower_components` directory with all the components we just installed. We can now make their Sass available for us to use with `@import` statements at the top of `main.scss`.
+You should now have a `node_modules` directory with all the components we just installed. We can now make their Sass available for us to use with `@import` statements at the top of `main.scss`.
 
 <pre><code class="o-syntax-highlight--diff">// main.scss
 
-+@import 'o-colors/main';
-+@import 'o-spacing/main';
-+@import 'o-typography/main';
-+@import 'o-buttons/main';
++@import '@financial-times/o-colors';
++@import '@financial-times/o-spacing';
++@import '@financial-times/o-typography';
++@import '@financial-times/o-buttons';
```

Comment:
We decided to not supply a `_index.scss` in the end, we need to update this code example to use the main.scss files
```suggestion
+@import '@financial-times/o-colors/main';
+@import '@financial-times/o-spacing/main';
+@import '@financial-times/o-typography/main';
+@import '@financial-times/o-buttons/main';
```